src/base/iana/rtype.rs

```diff
@@ -425,3 +425,13 @@ int_enum! {
 }
 
 int_enum_str_with_prefix!(Rtype, "TYPE", b"TYPE", u16, "unknown record type");
+
+pub trait IsGlue {
```

Comment:
What’s the reason to have this as  a trait rather than just a normal method?

Reply:
Errm, none? I saw a macro and avoided it ;-)

Reply:
Done.